tools/autograd/gen_python_functions.py

```diff
@@ -35,6 +35,7 @@
     'nonzero(_(out|numpy))?',
     'set_quantizer_',
     'set_data',
+    '.*_overrideable',  # overrideable functions for backend extension
```

Comment:
I guess not!

Reply:
Yup :D It bit me really hard and then I realize name of function matter haha ;)